rll/automate_upgrade.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Add entry in /etc/cron.d/ to daily check and excute an upgrade for rightlink.
+
+# -e will immediatly exit out of script at point of error
+set -e
+
+cron_file='/etc/cron.d/rightlink_upgrade'
+exec_file='/usr/local/bin/rightlink_check_upgrade'
+
+# Grab toggle option to disable
+if [[ "$DISABLE_AUTO_UPGRADE" == 'true' ]]; then
+  if [ -e ${cron_file} ]; then
+    rm -f ${cron_file}
+    echo "Automatic upgrade disabled"
+  else
+    echo "Automatic upgrade never enabled - no action done"
+  fi
+else
+  # If cron file already exists, will recreate it and update cron config with new random times.
+  [ -e ${cron_file} ] && echo "Recreating cron entry"
+
+  # Generate executable script to run by cron
+  cat > ${exec_file} << 'EOF'
```

Comment:
Added here for being a single point of management.  Putting it in as an attachment in a RightScript and then updating the attachment file for cookbook use is a good amount of overhead for maintenance.